feature/experimental/telemetry_only/feature.textproto

```diff
@@ -17,9 +17,68 @@ id {
   version: 1
 }
 
+# Fan
 telemetry_path {
   path: "/components/component/fan/state/speed"
 }
+
```

Comment:
I presume we will be moving all the paths from this to a proper feature profile in a different set of PRs?

Reply:
Yes we have more component paths to be addressed. i will move it together in single PR followed by this